iroh-node-util/src/rpc/client.rs

```diff
@@ -0,0 +1,19 @@
+use anyhow::Result;
+use futures_lite::{Stream, StreamExt};
+
+pub mod net;
+pub mod node;
+
+fn flatten<T, E1, E2>(
```

Comment:
maybe add this to quic-rpc? its used everywhere 🤣

Reply:
Where is this used? In the context of streaming responses, right? I can maybe add it somewhere in client as an util...

Reply:
Not super urgent, but I am adding it to quic-rpc: https://github.com/n0-computer/quic-rpc/pull/120